ddtrace/contrib/integration_registry/_registry_schema.json

```diff
@@ -22,17 +22,25 @@
         "type": "object",
         "description": "Min, max, and list of tested versions for a specific dependency.",
         "properties": {
-          "min": {
-            "description": "Minimum explicit version found in test lockfiles.",
+          "tested_min": {
+            "description": "Minimum version found in test lockfiles.",
             "type": "string"
           },
-          "max": {
-            "description": "Maximum explicit version found in test lockfiles.",
+          "tested_max": {
+            "description": "Maximum version found in test lockfiles.",
+            "type": "string"
+          },
+          "supported_min": {
+            "description": "Minimum implied version supported by the integration (Major version was tested).",
```

Comment:
It's not clear what is meant by "implied" here. I'm also having trouble parsing this sentence.